content/english/python-tensorflow/wrapup.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Workshop Conclusion"
+date: 2020-09-09T16:19:17-07:00
+prereq: "Python Basics: Functions- Built in Functions, Functions from libraries; Data Types- Strings, Numbers, Reading from Console; Data Structures- Lists, Tuples, Sets; Loops- For Loops"
+difficulty: Intermediate
+weight: 14
+draft: false
+---
+
+
+#### Congratulations! 
+
+You and your colleagues are now able to sort 4 times as many returned clothes! Your supervisor is impressed by this and chooses you as the Employee of the Month!
+
+Thank you for doing this workshop! We hope you enjoyed it and learned the basics of Machine Learning!
+
+{{% notice warning %}}
+#### Are you stuck anywhere in this workshop or want to check your work? 
+Feel free to refer to this: <a href="#" target="_blank">Answer Key</a>
+
+{{% /notice %}}
```

Comment:
We don't usually provide answers to the end of our workshops, but this is up for discussion.